### PR TITLE
Update status-react quickstart and build-status

### PR DESCRIPTION
**Pull request by jinhojang6**
1. Quick start

Update outdated instructions at https://status.im/technical/build_status/status_react_quickstart.html

- Replace `make startdev-ios-simulator` with `make run-clojure` (+ a screenshot)
- Replace `make react-native-ios` with `make run-metro` (+ a screenshot)
- Removed outdated screenshots
- Removed the `Debugging & Developing with re-frisk` section as `http://localhost:4567` doesn't work
- Replace `MacOS 10.15 “Catalina” or later` with `Nix`

Test link: https://dev-lang.status.im/technical/build_status/status_react_quickstart.html

<br/>

2. Build Status
- Replace `Prompt will show when Figwheel connects to your application` with `Build completed` as I can find in the terminal.

Test link: https://dev-lang.status.im/technical/build_status/

### CODE REVIEW

File: source/technical/build_status/status_react_quickstart.md [hunk 1]
```diff
@@ -111,4 +94,4 @@ make nix-purge
 
 # Known Issues
 
-[MacOS 10.15 "Catalina" or later](https://github.com/status-im/status-react/blob/develop/nix/README.md#macos-1015-catalina)
+- [Nix](https://github.com/status-im/status-react/blob/develop/nix/README.md#macos-1015-catalina)
```
Comment (jakubgs) [inline source/technical/build_status/status_react_quickstart.md:97]: I'm not sure this change makes sense. The issue is related to Nix, yes, but only on MacOS 10.15 or higher, so the title was pretty accurate if you ask me. Maybe you could prefix it with `Nix on`.

**Review by jakubgs — CHANGES_REQUESTED** [review]
I see you remove some outdated screenshots from the markdown file, but not from the repo. Why?

File: source/technical/build_status/status_react_quickstart.md (hunk 1 above)
Comment (jinhojang6) [reply source/technical/build_status/status_react_quickstart.md:97]: Didn't know that. Let me update it to `[Nix on MacOS 10.15 or higher](https://github.com/status-im/status-react/blob/develop/nix/README.md#macos-1015-catalina)`

**Comment by jinhojang6** [comment]
Thought it would be better to keep them as they could be used somewhere. Let me delete the outdated png files.

**Comment by jinhojang6** [comment]
@jakubgs Updated the commit based on your feedback. 

https://dev-lang.status.im/technical/build_status/status_react_quickstart.html

**Review by jakubgs — APPROVED** [review]
LGTM, thanks for fixing.

**Pull request merged**